front/app/api/phase_files/useDeletePhaseFile.ts

```diff
@@ -19,14 +19,16 @@ const deletePhaseFile = ({
 const useDeletePhaseFile = () => {
   const queryClient = useQueryClient();
 
-  return useMutation({
-    mutationFn: deletePhaseFile,
+  return useMutation<unknown, unknown, DeletePhaseFileArgs>({
```

Comment:
Might be better to use stricter types here
```suggestion
  return useMutation<Omit<BaseResponseData, "included">, CLErrors, DeletePhaseFileArgs>(
```

Not a huge fan of `<Omit<BaseResponseData, "included">` but it can do for now. Else we should specify the return type